src/java/picard/vcf/SortVcf.java

```diff
@@ -34,7 +34,17 @@
 @CommandLineProgramProperties(
         usage = "Sorts one or more VCF files according to the order of the contigs in the header/sequence dictionary and then by coordinate. " +
         "Can accept an external sequence dictionary. If no external dictionary is supplied, multiple inputs' headers must have " +
```

Comment:
> multiple inputs' headers

Avoid this form like the plague. Rephrase to e.g. "If no external dictionary is supplied and multiple inputs are provided, their headers must have (...)"

Actually, bonus points for rephrasing to an active rather than passive voice.